images/Makefile

```diff
@@ -0,0 +1,45 @@
+# Container registry (default to dockerhub) 
+REGISTRY ?= registry.hub.docker.com/library
```

Comment:
Why is the default registry dockerhub when CI pushes them to github registry?

Reply:
Because I don't want to accidentally push container image updates to our GCR registry when running `make docker-build`. It almost happened to me twice, it may happen to others.